src/lib/webServer.ts

```diff
@@ -133,7 +133,7 @@ export class WebServer {
                                 // This is very bad, and perhaps the adapter should also terminate itself?
                             }
                         }
-                        // TODO: How new certificates will be used?
+                        // contexts is now up to date and will be utilised in SNICallback - nothing more to do.
```

Comment:
would just remove it entirely

Reply:
... and then have someone else come along and wonder how it works? Doesn't hurt to explain.

Reply:
Ok can also leave it there maybe it’s just not my comment style to have them at the end of code, but fine for me here